brianlong/www.validators.app
# 186268962 gather solana tps stats

#### What's this PR do?
- gather TPS stats in PingThingStats
- apply those data to a chart

#### How should this be manually tested?
- run `rails r dev/keep_creating_ping_things.rb`
- start sidekiq
- check the charts in ping things page if it's populated correctly
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186268962)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
